workspaces/tech-radar/plugins/tech-radar/README.md

```diff
@@ -197,7 +197,7 @@ The TS example can be found [here](src/sample.ts).
 
 ### How do I load in my own data?
 
-The `TechRadar` plugin uses the `techRadarApiRef` to get a client which implements the `TechRadarApi` interface. The default sample one is located [here](https://github.com/backstage/backstage/blob/master/plugins/tech-radar/src/sample.ts). To load your own data, you'll need to provide a class that implements the `TechRadarApi` and override the `techRadarApiRef` in the `app/src/apis.ts`.
+The `TechRadar` plugin uses the `techRadarApiRef` to get a client which implements the `TechRadarApi` interface. The default sample one is located [here](src/sample.ts). To load your own data, you'll need to provide a class that implements the `TechRadarApi` and override the `techRadarApiRef` in the `app/src/apis.ts`.
```

Comment:
I'm not sure the relative path works when browsing the README on npm
```suggestion
The `TechRadar` plugin uses the `techRadarApiRef` to get a client which implements the `TechRadarApi` interface. The default sample one is located [here](https://github.com/backstage/community-plugins/blob/main/workspaces/tech-radar/plugins/tech-radar/src/sample.ts). To load your own data, you'll need to provide a class that implements the `TechRadarApi` and override the `techRadarApiRef` in your `app/src/apis.ts`.
```

Reply:
I just copied the existing link to the samples further up in the file; https://github.com/backstage/community-plugins/pull/572/files#diff-910dec72e958883cd5bf17dbc5dfc2aeb9e514118f1ba65850484cb28a9cae01R119

It appears to link fine in the npm page (https://www.npmjs.com/package/@backstage-community/plugin-tech-radar) so I expect this would work as well?

Reply:
ah amazing, thx for checking!